src/odemis/driver/_mccdaq.py

```diff
@@ -0,0 +1,674 @@
+        input will proceed as configured.  The EXTTRIG option must be used
+        in the AInScan command to utilize this feature.
+
+         type:  the type of trigger  (0 = external trigger falling edge, 1 = external trigger rising edge)
+         chan:  the digial bit that receives the trigger signal(0 for USB-120LS, 0-3 for miniLAB 1008)
+        '''
+        self.h.write([self.SET_TRIGGER, type, chan, 0, 0, 0, 0, 0, 0])
+
+    def SetID(self, id):
+        '''
+        This command stores an identifier on the unit.  Values of 0-255 are valid.  Note that 0
+        is used to flag uninitialized units.
+
+        id: user id number (0-255)
+        '''
+
+        if id < 0 or id > 255:
+            raise ValueError('SetID: id out of range')
+
+        self.h.write([self.SET_ID, id, 0, 0, 0, 0, 0, 0])
+
+    def GetID(self):
+        '''
+        This function retrieves the id number stored on the device.  Note that a value of 0
+        is used to flag an uninitialized device.
+        '''
+
+        self.h.write([self.GET_ID, 0, 0, 0, 0, 0, 0, 0])
+
+        value = self.h.read(8, 100)
+
+        return (value[0])
+
+    def volts(self, gain, value):
+        '''
+        converts raw values to volts
+        '''
+        volt = 0.0
+        if gain == self.SE_10_00V:
+            volt = value * 10.0 / 0x3ff
+        elif gain == self.BP_20_00V:
+            volt = value * 20.0 / 0x7ff
+        elif gain == self.BP_10_00V:
+            volt = value * 10.0 / 0x7ff
+        elif gain == self.BP_5_00V:
+            volt = value * 5.0 / 0x7ff
+        elif gain == self.BP_4_00V:
+            volt = value * 4.0 / 0x7ff
+        elif gain == self.BP_2_50V:
+            volt = value * 2.5 / 0x7ff
+        elif gain == self.BP_2_00V:
+            volt = value * 2.0 / 0x7ff
+        elif gain == self.BP_1_25V:
+            volt = value * 1.25 / 0x7ff
+        elif gain == self.BP_1_00V:
+            volt = value * 1.0 / 0x7ff
+        else:
+            raise ValueError('volts: Unknown range.')
+
+        return volt
+
+
+class usb_1208LS(usb_1208):
+    def __init__(self, serial=None):
+        self.productID = 0x0007a  # USB-1208LS
+        usb_1208.__init__(self)
+        try:
+            logging.info(f"Trying opening device USB-1208LS with s/n {serial}..")
+            self.h.open(0x09db, self.productID, serial)
+        except Exception as ex:
+            raise HwError(f"Failed to find USB device {self.productID}, serial {serial}: {ex}")
+
+        self.device_type, self.device_sn = self.h.get_product_string(), self.h.get_serial_number_string()
+        logging.debug(f"Successfully connected with device type {self.device_type} with s/n {self.device_sn}")
+
+        # enable non-blocking mode
+        self.h.set_nonblocking(1)
+
+        # set default configuration
+        self.DConfig(self.DIO_PORTA, 0x00)  # Port A output
+        self.DConfig(self.DIO_PORTB, 0x00)  # Port B input
```

Comment:
The comments don't seem to match the code. Both ports are set to "input" (0) at init, right?